LinuxComputeVM/install-services

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+install_service () {
+    echo "Installing service: $1..."
```

Comment:
Why not if [ ! -f $1]; then exit 1 in first place ? I think it will simplify code a bit.

Reply:
I agree.